kitchen.dokken.yml

```diff
@@ -43,15 +38,10 @@ platforms:
     intermediate_instructions:
       - RUN /usr/bin/apt-get update
 
-- name: centos-6
```

Comment:
We want to keep centos-6 testing since it's still heavily used in enterprise environments and supported until Nov

Reply:
added centos-6 back